pkg/runtime/reconciler.go

```diff
@@ -1064,6 +1042,50 @@ func (r *resourceReconciler) getEndpointURL(
 	return r.cfg.EndpointURL
 }
 
+// getResyncPeriod returns the duration a reconciler should use to drift remediate custom
+// resources. It attempts to retrieve the duration from the following sources, in this order:
+// 1. A resource-specific drift remediation period specified in the drift remediation
+//    configuration JSON.
+// 2. A resource-specific requeue on success period specified by the resource manager factory.
+// 3. The default drift remediation period specified in the configuration.
+// 4. The default resync period defined in the ACK runtime package.
+//
+// Each reconciler has a unique value to use. This function should only be called during the
+// instantiation of an AWSResourceReconciler and should not be called during the reconciliation
```

Comment:
Be more specific here:  instead of "the reconciliation function", give a reference to the specific function.